src/mds/agg_mds/adapters.py

```diff
@@ -1592,6 +1605,8 @@ def gather_metadata(
             globalFieldFilters=globalFieldFilters,
             schema=schema,
         )
+        print("Result after normalizing: ")
+        print(results)
```

Comment:
why adding all these `print()` in this file? Are they for debugging? If so, please changes them to use `logger.debug()`